resource/ckb.toml

```diff
@@ -7,11 +7,11 @@ data_dir = "data"
 
 [chain]
 # Choose the kind of chains to run, possible values:
-# - specs/dev.toml
-# - specs/testnet.toml
-spec = "specs/dev.toml" # {{
-# testnet => spec = "specs/testnet.toml"
-# integration => spec = "specs/integration.toml"
+# - { file = "specs/dev.toml" }
+# - { bundled = "specs/testnet.toml" }
```

Comment:
`bundled` is hard to understand in config, can we use some intention related name like `builtin`.